scripts/build/Platform/Linux/build_config.json

```diff
@@ -195,7 +195,7 @@
       "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DLY_UNITY_BUILD=FALSE -DLY_PARALLEL_LINK_JOBS=4",
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "AssetProcessorBatch",
-      "ASSET_PROCESSOR_BINARY": "bin/profile/AssetProcessorBatch",
+      "ASSET_PROCESSOR_BINARY": "bin/profile/AssetProcessorBatch --ignoreFutureAssetDatabaseVersionError",
```

Comment:
I think this is on the wrong line

Reply:
+1 it should probably be part of ASSET_PROCESSOR_OPTIONS

Reply:
Good catch, fixed